docs/DeveloperGuide.md

```diff
@@ -287,6 +287,101 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
+**Use case: Add a tag**
+
+**MSS**
+
+1. User requests to add a new tag.
+2. StonksBook add the provided tag.
+
+**Extensions**
+
+* 2a. The provided tag already exists in the tag list.
+
+Use case ends.
```

Comment:
Can this be indented to be the same level as `2a`? Likewise for the remaining use cases